etc/cdn-log-shipper/log-shipper.yml

```diff
@@ -146,6 +157,12 @@ Resources:
             // podcast id and episode guid (only works for dovetail3-cdn requests)
             const datas = mappedRows.filter(data => {
               const parts = data['cs-uri-stem'].split('/').filter(s => s);
+
+              // if the path starts with a region like usw2, shift that off
+              if (parts[0] && parts[0].match(/^[a-z][a-z0-9\-]+$/)) {
```

Comment:
This was the other bug, that we have requests with an aws region name prefix, like `/usw2/`
These were all getting filtered out